src/mbgl/gl/context.cpp

```diff
@@ -124,10 +132,23 @@ void Context::initializeExtensions(const std::function<gl::ProcAddress(const cha
             return nullptr;
         };
 
-        debugging = std::make_unique<extension::Debugging>(fn);
-        if (!disableVAOExtension) {
-            vertexArray = std::make_unique<extension::VertexArray>(fn);
-        }
+        const std::string renderer = reinterpret_cast<const char*>(MBGL_CHECK_ERROR(glGetString(GL_RENDERER)));
+        Log::Info(Event::General, "GPU Identifier: %s", renderer.c_str());
+
+        debugging = initializeExtension<extension::Debugging>(fn, [&renderer]() {
+            // Blacklist ANGLE since the debugging extension is causing crashes
```

Comment:
While we're touching this code, let's stop using the word "blacklist" and replace it with something like "blocklist", or "stop list".

Reply:
Sure, will do.